Application/EdFi.Ods.Extensions.TPDM/Artifacts/Metadata/InterchangeOrderMetadata-EXTENSION.xml

```diff
@@ -60,9 +60,6 @@
     <Element name="InterventionClassDescriptor" />
     <Element name="InteractivityStyleDescriptor" />
     <Element name="InstitutionTelephoneNumberTypeDescriptor" />
-    <Element name="IndicatorLevelDescriptor" />
```

Comment:
These shouldn't be getting removed.  It looks like you need to bump up to MetaEd 2.2.1-dev.7 to get the latest core model since core affects this output file, or just drop the changes to this file as a simpler change.